GameServerLib/Logic/GameObjects/Unit.cs

```diff
@@ -133,17 +133,29 @@ public string Model
 
         {
             BuffGameScriptControllers = new List<BuffGameScriptController>();
-            this.stats = stats;
+            stats = s;
```

Comment:
How about renaming "stats" to "Stats" and Stats in the constructor to "stats" so it is in the proper naming convension ?
I think it would improve on clarity.

Reply:
Great idea, it's on its way